test_scripts/RC/Capabilities/018_ControlCapabilities_Empty_array_RADIO.lua

```diff
@@ -0,0 +1,51 @@
+---------------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SDL_RC] Capabilities
+--
+-- Description:
+-- In case:
+-- HMI respond with empty array for module to RC.GetCapabilities request from SDL
+--
+-- SDL must:
+-- Reject all RPCs from mobile application to such module with result code UNSUPPORTED_RESOURCE, success:false
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonRC = require('test_scripts/RC/commonRC')
+
+--[[ Local Variables ]]
+local disabledModule = "RADIO"
+local enabledModule = "CLIMATE"
```

Comment:
@Itileda These variables are introduced in order to simplify creating of a few test scripts because the flow is the same